spec/support/shared_examples_for_idv_account_creation.rb

```diff
@@ -0,0 +1,149 @@
+shared_examples 'idv account creation' do |sp|
```

Comment:
Any concerns about the majority of the actual test code piling up in `spec/support` with the `shared_examples_` prefix? I'm looking there now and it already looks crowded. What do you think about moving this to something like `spec/support/idv_examples/account_creation.rb`?